quinn-h3/src/proto/settings.rs

```diff
@@ -19,6 +19,15 @@ macro_rules! setting_identifiers {
 
 impl SettingId {
     const NONE: SettingId = SettingId(0);
+
+    fn is_supported(id: SettingId) -> bool {
```

Comment:
```suggestion
    fn is_supported(self) -> bool {
```

This would be a bit more convenient to call.